querydsl-sql/pom.xml

```diff
@@ -21,7 +21,7 @@
       org.joda.time.*;version="[1.6,3)",
       ${osgi.import.package.root}
     </osgi.import.package>
-
+    <surefire.argLine/>
```

Comment:
Hmm, I see that Maven complains about a recursive definition, I'll have to think about a solution.